src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -148,6 +148,30 @@ public void nonexistentFunctions() throws Exception {
         r.assertLogContains("constructible with compass and straightedge", b);
     }
 
+    /**
+     * Tests the ability to execute a step with an unnamed monomorphic describable argument.
+     */
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphic() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        p.setDefinition(new CpsFlowDefinition("monomorphStep firstArg:'one', secondArg:'two'"));
```

Comment:
Anyway I do not think such a patch to `DSL` is even necessary for usability, since you can already add a `@Symbol` to the nested struct and then mark the step as a metastep and get similar syntax:

```groovy
monomorphStep monomorph(firstArg: 'one', secondArg: 'two')
```

can become simply

```groovy
monomorph firstArg: 'one', secondArg: 'two'
```

as of #28. Which reinforces my request to see analogous tests for `@Symbol`ic structs.

@oleg-nenashev’s original example with `PublishHTMLStep` is a perfect example of this. We would want to make this a metastep, add `@Symbol("publishHTML")` to `HtmlPublisherTarget.DescriptorImpl`, make `HtmlPublisherTarget` use `@DataBoundSetter` properly (leaving only `reportDir` in the `@DataBoundConstructor`), and make sure that `DSL` and `Snippetizer` understand that there is no real ambiguity in

```groovy
publishHTML 'target/javadoc'
```